test/test-eth-trezor-keyring.js

```diff
@@ -319,39 +319,53 @@ describe('TrezorKeyring', function () {
     })
 
     describe('signMessage', function () {
-        it('should throw an error because it is not supported', function () {
-            expect(_ => {
-                keyring.signMessage()
-            }).to.throw('Not supported on this device')
+        it('should call TrezorConnect.ethereumSignMessage', function (done) {
+            const sandbox = chai.spy.sandbox()
+            sandbox.on(TrezorConnect, 'ethereumSignMessage')
+            keyring.signMessage(fakeAccounts[0], 'some msg').catch(e => {
+                // we expect this to be rejected because
+                // we are trying to open a popup from node
+                expect(TrezorConnect.ethereumSignMessage).to.have.been.called()
+                sandbox.restore()
+                done()
+            })
         })
     })
 
     describe('signPersonalMessage', function () {
         it('should call TrezorConnect.ethereumSignMessage', function (done) {
 
-            chai.spy.on(TrezorConnect, 'ethereumSignMessage')
+            const sandbox = chai.spy.sandbox()
+            sandbox.on(TrezorConnect, 'ethereumSignMessage')
             keyring.signPersonalMessage(fakeAccounts[0], 'some msg').catch(e => {
                 // we expect this to be rejected because
                 // we are trying to open a popup from node
                 expect(TrezorConnect.ethereumSignMessage).to.have.been.called()
+                sandbox.restore()
                 done()
             })
         })
     })
 
     describe('signTypedData', function () {
-        it('should throw an error because it is not supported', function () {
-            expect(_ => {
-                keyring.signTypedData()
-            }).to.throw('Not supported on this device')
+        it('should throw an error because it is not supported', async function () {
+            try {
+                await keyring.signTypedData()
+            } catch (e) {
```

Comment:
why didn't you use `throw` ? I believe this test is going to fails if is not throwing anything. This shouldn't be the case now but if something change it could happen.

Reply:
The test is specifically making sure it throws

Reply:
I mean, is not going to catch any error. If this tests wants to catch an error and `signTypedData` doesn't throw this test won't `expect()`  anything, it will simply skip the catch condition.

Reply:
Oh got it. I'll fix it.